Clean up place name rendering

math1985 opened the pull request:
* Don't render place=town on z16 and higher, and place=suburb and place=village
  on z17 and higher.
* Render place=hamlet, place=locality, place=neighbourhood,
  place=isolated_dwelling, and place=farm from z15 rather than from z14 (resolves
  #948).
* Render place=town black rather than gray.
* Make font sizes more consistent by increasing font size for place=town on z12
  and z13, for place=suburb on z16, and for place=village on z14 and z16
  (resolves #734, resolves #1204).
* Treat capital=4 like capital=yes (resolves #522).
* Make text-wrap consistent.

pnorman commented:
I like the other changes (in principle, haven't previewed yet) but I'm not sure about adding the new tags. In particular, borough was rejected (#954)

https://github.com/gravitystorm/openstreetmap-carto/issues/954#issuecomment-55747887
> "borough" and "municipality" are both, in British English, terms for administrative levels, not places. They are not in the hierarchy of hamlet/village/town/city, nor are they in the similar set of place types suburb/metropolis/conurbation etc.

> I've no idea how they came into being in the place hierarchy on the wiki instead of either a) being part of the admin_level hierarchy or b) better values, related to places, being chosen.

> I suspect this is just adding more tags for dubious overall value. Having a limited set of place tags provides *more* value than having an large number of place tags that are hard if not impossible to choose between.

> If a village is also a municipality, then that doesn't mean it stops being a village. If a suburb has a local council, then it's still a suburb. The administrative classification should be recorded separately.

math1985 commented:
Thanks, I will remove borough. What about the other new tags?

pnorman commented:
I'm skeptical - but haven't looked into them in any detail. I'm also not a native en_GB speaker, only en_CA.

We do not want to encourage a proliferation of place values as that reduces their value for data consumers

math1985 commented:
* About 60% of the place=quarter occurrences are in Japan.
* About half the place=city_block occurrences are in Stockholm.
* About 99% of the place=block occurrences are in Malmo, Lund and Landskrona.

Block names seem rather particular to Sweden, but they seem to be important toponyms there.

math1985 commented:
So, should I drop quarter/block/city_block from this PR?

math1985 commented:
> 

So, should I drop quarter/block/city_block from this PR?

math1985 commented:
> So, should I drop quarter/block/city_block from this PR?

Any opinions on this?

pnorman commented:
I'd drop the new tags from the PR.

math1985 commented:
> I'd drop the new tags from the PR.

Dropped the tags and rebased.

Do we consider the proposal to add rendering for quarter/block/city_block as 'rejected'?

gravitystorm commented:
> Do we consider the proposal to add rendering for quarter/block/city_block as 'rejected'?

Yes, no more place name values please.

dieterdreist commented:
2015-02-04 14:12 GMT+01:00 Andy Allan <notifications@github.com>:

> Do we consider the proposal to add rendering for quarter/block/city_block
> as 'rejected'?
>
> Yes, no more place name values please.
>


IMHO there is a point in having at least some of them for places where they
are used (probably for good reason). E.g. in the centre of Mannheim there
are blocks but only street names for some principal road:
http://de.wikipedia.org/wiki/Quadratestadt

In other cases, "suburb" and "neighbourhood" are not sufficient to
represent the hierarchy of toponyms (not talking about administrative units
here), e.g. Rome. (We have not yet mapped all those toponyms that are in
use, but we will sooner or later do it). That's where the proposal to have
also a place=quarter tag comes from.

Also, using "suburb" for stuff that is not a "suburb" is neither intuitive
nor semantically correct, so despite all the historical reasons it wouldn't
be too bad IMHO to change this sooner or later to something more pertinent.

If the main style decides to ignore these tags that are at least in some
cases there for good reason (the established tags don't hit it), we will
encourage people to map with tags that don't apply (tagging for the
renderer) in order to retain the usability of the map.

RAytoun commented:
Many old towns have historic areas such as "old quarter", which are still  in common use and named on town and tourist maps as a place to visit. These areas do not necessarily coincide with administrative boundaries and do not have administrative status.

pnorman commented on placenames.mss:
```diff
@@ -84,51 +84,57 @@
   }
 }
 
 #placenames-small::suburb {
```
We probably want to move these out of their own attachments and do an order by on the query - but out of scope of this PR

math1985 replied:
@pnorman Is there an advantage to that? Performance?

kocio-pl commented:
> In other cases, "suburb" and "neighbourhood" are not sufficient to represent the hierarchy of toponyms (not talking about administrative units here), e.g. Rome. (We have not yet mapped all those toponyms that are in use, but we will sooner or later do it). That's where the proposal to have also a place=quarter tag comes from. 

That's also the case in Warsaw. We have three levels deep place naming hierarchy (suburb/quarter/neighbourhood - and it's not administrative too) and quarters are not easily negligible because its name is visible on every house number and city name plate, so it's used as a main navigation unit:

https://pl.wikipedia.org/wiki/Miejski_System_Informacji_%28Warszawa%29#mediaviewer/File:MSI_Trakt_Kr%C3%B3lewski.jpg
https://pl.wikipedia.org/wiki/Miejski_System_Informacji_%28Warszawa%29#mediaviewer/File:Tablica_MSI_ulica_Alternatywy_w_Warszawie.JPG
